src/Components/Onboarding/Steps/Artists/index.tsx

```diff
@@ -32,7 +32,9 @@ export default class Artists extends React.Component<StepProps, State> {
   }
 
   updateFollowCount(count: number) {
-    this.setState({ followCount: count })
+    const updatedFollowCount = this.state.followCount + count
```

Comment:
@yuki I believe this addresses the edge cases you outline [here](https://github.com/artsy/reaction/pull/464#pullrequestreview-89919235).

Reply:
Did you mean @yuki24?

Reply:
😆 🤣 😆 🤣 #DidYouMean